contents/docs/experiments/holdouts.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Holdouts
+---
+
+Holdouts let you create persistent groups of users who can be excluded from multiple experiments or all experiments.
```

Comment:
```suggestion
When running A/B tests, you can create permanent lists of users called "holdouts" who won't participate in experiments. You can keep these same users out of either specific experiments or all of your testing entirely.
```

Reply:
question actually: Can you specify which users in the holdout? Or is it random?

Reply:
We now call it "Experiments", so let's use "When running experiments, ..."

Reply:
@Lior539 Why use "permanent" instead of "persistent"? I think "persistent" fits better here.

To me, "permanent" suggests something that never changes. But holdouts can, and should, be lifted after a certain period. They persist across multiple experiments, but they aren't permanent because, at some point, the holdout will no longer be used.

Reply:
you're right @jurajmajerik ! I think we can actually  remove the word "permanent/persistence" altogether. The sentence still makes sense then.

Also flagging in case it got missed:
> Can you specify which users are in the holdout? Or is it random?

Reply:
Applied a couple of edits 84dcf6042c017149185d3498321612d1bfbc0032, f6e682cc38a22d703b3614db0fe7013c17a036ee